test/fakeintake/server/server.go

```diff
@@ -361,7 +371,7 @@ func (fi *Server) handleGetPayloads(w http.ResponseWriter, req *http.Request) {
 		}
 		jsonResp, err = json.Marshal(resp)
 	} else if serverstore.IsRouteHandled(route) {
-		payloads, payloadErr := fi.store.GetJSONPayloads(route)
+		payloads, payloadErr := serverstore.GetJSONPayloads(fi.store, route)
```

Comment:
Why don't you keep `GetJSONPayloads` as a method on the store interface?

Reply:
I preferred to split it because:
- The store itself is only used to retrieve the raw payloads so I think it can be useful to separate the concerns.
- It's more generic
What do you think ?